src/api-clients/room-api-client.js

```diff
@@ -36,6 +36,15 @@ class RoomApiClient {
       )
       .then(res => res.data);
   }
+
+  deleteRoom(roomId) {
+    return this.httpClient
+      .delete(
+        `/api/v1/rooms/${roomId}`,
```

Comment:
Let's keep consistent and `encodeURIComponent(roomId)`, since we did this now in all api clients.

Reply:
Done